docker/Dockerfile

```diff
@@ -14,14 +14,21 @@ RUN DEBIAN_FRONTEND=noninteractive /tmp/set_up_ubuntu.sh
 
 
 FROM socorro_image_base as socorro_breakpad
+
 WORKDIR /mdsw/
 
+# Install some helpful debugging things
+RUN apt-get -y install gdb vim
```

Comment:
This will only exist in this intermediary container and it doesn't make it into the final socorro_app image that we use in stage and prod.